lib/matplotlib/backend_bases.py

```diff
@@ -1503,6 +1512,17 @@ def __init__(self, name, canvas, key, x=0, y=0, guiEvent=None):
         super().__init__(name, canvas, x, y, guiEvent=guiEvent)
         self.key = key
 
+    @classmethod
+    def _from_ax_coords(cls, name, ax, xy, key, *args, **kwargs):
+        """Generate a synthetic event at a given axes coordinate."""
+        # Separate from MouseEvent._from_ax_coords instead of being defined in the base
+        # class, due to different parameter order in the constructor signature.
+        x, y = ax.transData.transform(xy)
+        event = cls(name, ax.figure.canvas, key, x, y, *args, **kwargs)
```

Comment:
Let's make the signature explicit. It's not worth employing `*args, **kwars` for a single parameter.

```suggestion
    def _from_ax_coords(cls, name, ax, xy, key, guiEvent=None):
        """
        Generate a synthetic event at a given axes coordinate.

        This is mainly intended for creating events during testing. The event can be send
        by calling its ``_process()`` method.
        """        
        # Separate from MouseEvent._from_ax_coords instead of being defined in the base
        # class, due to different parameter order in the constructor signature.
        x, y = ax.transData.transform(xy)
        event = cls(name, ax.figure.canvas, key, x, y, guiEvent)
```

Reply:
done